### PR TITLE
Add RichTextEditor and PanelActions tests

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

Adds some tests around the RichTextEditor and PanelActions, specifically: 
- Tests the roving index (tab and arrow navigation) of PanelActions
- Tests the basic rendering of RichTextEditor
- Tests the aria-pressed states of the RichTextEditor toolbar buttons
- Tests the roving index (tab and arrow navigation) of RichTextEditor

### CODE REVIEW

File: components/form-builder/app/edit/elements/lexical-editor/tests/RichTextEditor.test.js [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+        <RichTextEditor
+          path="path.to.content"
+          content="Here is some test content"
+          autoFocusEditor={true}
+          ariaLabel="AriaLabel"
+        />
+      </Providers>
+    );
+
+    // rendered.debug();
```
Comment (timarney) [inline components/form-builder/app/edit/elements/lexical-editor/tests/RichTextEditor.test.js:27]: Should remove this

File: components/form-builder/app/edit/elements/lexical-editor/tests/RichTextEditor.test.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
```
Comment (timarney) [inline components/form-builder/app/edit/elements/lexical-editor/tests/RichTextEditor.test.js:75]: should remove this

File: components/form-builder/app/edit/elements/lexical-editor/tests/RichTextEditor.test.js [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
```
Comment (timarney) [inline components/form-builder/app/edit/elements/lexical-editor/tests/RichTextEditor.test.js:143]: Should remove this

**Comment by dsamojlenko** [comment]
@timarney

**Review by timarney — APPROVED** [review]

**Pull request merged**